### PR TITLE
Better test run

### PR DESCRIPTION
**Pull request by nscarcella**
This update improves the sanity/examples runners. We now use mocha to run the wollok tests, which provides a much better presentation of the runs and displays the duration of tests that took too long.

It also now shows the list of failed tests on the bottom and  allows test filtering by the file, package or test description (it's done in the same way that all other tests).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,22 +26,21 @@
...
     "chai": "^4.2.0",
-    "chai-as-promised": "^7.1.1",
     "chalk": "^2.4.2",
-    "command-line-args": "^5.1.1",
     "globby": "^10.0.1",
     "mocha": "^6.2.2",
-    "simple-git": "^1.128.0",
-    "sinon": "^7.5.0",
+    "simple-git": "^1.129.0",
+    "sinon": "7.3.2",
```
Comment (fdodino) [inline package.json]: did you need to downgrade sinon version?

File: src/interpreter.ts [hunk 2]
```diff
@@ -705,8 +707,54 @@ function run(evaluation: Evaluation, natives: Natives, sentences: List<Sentence>
...
+    run(evaluation, natives, test.body.sentences)
+  } catch (e) {
+    error = e
+  }
+  const delta = process.hrtime(before)
+
+  return {
+    evaluation,
+    error,
+    duration: round(delta[0] * 1e3 + delta[1] / 1e6),
```
Comment (fdodino) [inline src/interpreter.ts]: I won't say anything about this magic calculus...

File: package.json (hunk 1 above)
Comment (nscarcella) [reply package.json]: Yes. We don't use sinon, it's just a transitive dependency and it was complaining that the version was out of range. Nothing big, but the warning bothered me.

File: src/interpreter.ts (hunk 2 above)
Comment (nscarcella) [reply src/interpreter.ts]: These monsters return a touple with seconds and nanoseconds. If you can think of a clearer way to translate it to ms I'm all ears

Comment (fdodino) [reply src/interpreter.ts]: I wonder why people with low-design skills take cards related with dates and times in core implementations (like java.util.Date, java.sql.Date, and also in JS).

I feel this is not the right moment, but maybe sometime we can use [momentjs simpler approach](https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/07-difference/).

Comment (fdodino) [reply src/interpreter.ts]: I'm not complaining about your solution, just for the record. I believe having a tuple with seconds and nanoseconds is obscure and forces you to remember that ugly interface for curious people like me.

Comment (nscarcella) [reply src/interpreter.ts]: Nah, I know. I´m not fond of it either...
Maybe there is some other interface somewhere to measure time in ms? I would rather not add a dependency just for this, though...

**Pull request merged**